roles/spark-standalone-service-install/meta/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+
+dependencies:
+  - role: spark-standalone-variables
+  - role: upstart
+    name: spark-dispatcher
```

Comment:
`spark-dispatcher` is a mesos-related service that is already created by the mesos version of the `spark` roles.  What you're likely looking for is `spark-master`.